eng/Version.Details.xml

```diff
@@ -36,5 +36,11 @@
       <Uri>https://github.com/dotnet/arcade</Uri>
       <Sha>eeb78d042d8f2bef222c83790fa253c5a22675ce</Sha>
     </Dependency>
+    <!-- Necessary for source-build. This allows the package to be retrieved from previously-source-built artifacts
+         and flow in as dependencies of the packages produced by msbuild. -->
+    <Dependency Name="System.Configuration.ConfigurationManager" Version="7.0.0">
```

Comment:
It seems like this should be considered a product dependency, right?

Reply:
Yes, I think you are right. Thanks for noticing, will move